feat(graphql): move schema to Github artifact

gligneul opened the pull request:
Close #77

gligneul committed:
fix: typo in CHANGELOG.md

gligneul committed:
feat(graphql): move schema to Github artifact

Remove the generated schema from the repository and generate it
automatically on the CI.

gligneul commented:
It works: https://github.com/cartesi/rollups-node/actions/runs/6239425867?pr=95

torives commented on .github/workflows/build.yml:
```diff
@@ -68,6 +68,16 @@ jobs:
+      - name: Generate GraphQL schema
+        run: ./target/debug/generate-schema
+
+      - name: Upload artifacts
```
```suggestion
      - name: Upload GraphQL schema
```
or just
```suggestion
      - name: Upload schema
```

gligneul committed:
feat: add changelog to the Github release

gligneul commented:
Fixed the issue. Now, it generates a proper release with the artifact. I also added the changelog to the release.
https://github.com/cartesi/rollups-node/releases/tag/v1.1.0-test.1
